### ISSUE TITLE
Remove or disable google analytics (by default)

### ISSUE THREAD

**Issue by NerdyProjects** [fightforthefuture/digital-climate-strike#76]
Hey there,

why do we want to support companies like google getting more user data?
We expect - or at least hope - that a lot of projects participate in the digital climate strike.

We

* piss off admins who see the google analytics by default
* give the data of users of websites, who previously didn't have google analytics, to google

and for what?...

**Comment by miloskroulik** [comment]
I agree completely. Perhaps we should switch GA from opt-out to opt-in?

**Comment by dtoakley** [comment]
Hey @NerdyProjects and @miloskroulik, 

First up, thank for this feedback. I'm aligned with you on not wanting to send user data to Google on websites that previously did not have Google Analytics. We thought about this default when we implemented this feature and settled on this default (with the option to easily remove it) for a number of reasons. I'll try to outline them as best I can here, along with what I recommend instead of changing the default:

**Comment by dtoakley** [comment]
Hey @NerdyProjects and @miloskroulik,

First up, thank for this feedback. I'm aligned with you on not wanting to send user data to Google on websites that previously did not have Google Analytics. We thought about this default when we implemented this feature and settled where we did for a number of reasons. I'll try to outline them as best I can, along with what I recommend instead of changing the default:

- As you’re probably aware, this javascript widget is part of a much broader campaign to drive turnout on September 20th and raise massive, widespread public awareness about the Global Climate Strike. A critical data point for generating media coverage, promoting the strikes, and for using to convince large website targets is the total number of websites that have added the widget. For example, we can now say publicly that over 900 sites have added the banner. Without GA being on by default, we would be unaware of the vast majority of sites and lose this critical tactic from our campaign toolbelt. 
- Ancillary to this, GA has enabled us to follow up some of the large sites and ask them a) to post about it on social media, and b) join the participants list of the website (https://digital.globalclimatestrike.net/#participants). Not only have the websites we’ve spoken been delighted by this, but added more logos on the website provides legitimacy to the campaign and our outreach has been strengthened. Without GA we would be unlikely to have this feedback loop and opportunity. 
- Providing the option to disable balances these benefits with the real issue you mention about Google Analytics being added (intentionally, or unintentionally) to a website that doesn’t want it. 

So, instead of flipping the default on this, I think we should update the README to be super clear (with some bolding, for example) that the widget adds Google analytics by default, but it can be disabled in the options. We could even include the specific additional script tag they will need so folks can just copy and paste it.

How does that sound? I totally hear y’all on the downside with this - but am also hoping this provides a bit more transparency / thinking into why it’s on by default.
